secretmanager/snippets/add_secret_version.py

```diff
@@ -39,25 +42,27 @@ def add_secret_version(project_id, secret_id, payload):
 
     # Convert the string payload into a bytes. This step can be omitted if you
     # pass in bytes instead of a str for the payload argument.
-    payload = payload.encode("UTF-8")
+    payload_bytes = payload.encode("UTF-8")
```

Comment:
mypy saw this as a type mismatch because of the self-assigning type change. Renaming the variable helps with clarity.